PublicationsRouter/v2/oai-pmh/VERBS.md

```diff
@@ -111,20 +133,41 @@ Returned information:
 * Records: notification metadata, only oai_dc metadata prefix is supported
 * Resumption Token: base64 encoded request parameters for next page
 
-PubRouter API request:
-
-Params:
-
-* since: provided "from" date, or earliest available datestamp
-* page: 1, or from resumptionToken
-* pageSize: 100
-
-The request is of the form:
-
-    GET /routed/<repo id>?<params>
-
-Note that this endpoint disregards any "until" or "set" parameters, as PubRouter has no equivalent concepts.
-
+Example request: `GET https://pubrouter.jisc.ac.uk/oaipmh/repo/123456789?verb=ListRecords&from=2017-01-01&metadataPrefix=oai_dc`
+
+Example response:
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<OAI-PMH xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.openarchives.org/OAI/2.0/" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd">
+    <responseDate>2018-05-23T14:26:53Z</responseDate>
+    <request verb="ListRecords" from="2017-01-01" metadataPrefix="oai_dc">http://pubrouter.jisc.ac.uk/repo/123456789</request>
+    <ListRecords>
+        <record>
+            <header xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+                <identifier>oai:pubrouter.jisc.ac.uk/notification:987654321</identifier>
+                <datestamp>2018-02-01T15:21:06Z</datestamp>
+            </header>
+            <metadata xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+            ... (for example of metadata go to XWALK.md)
+            </metadata>
+        </record>
+        <record>
+            <header xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+                <identifier>oai:pubrouter.jisc.ac.uk/notification:987654322</identifier>
+                <datestamp>2018-02-01T15:21:06Z</datestamp>
+            </header>
+            <metadata xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/">
+            ... (for example of metadata go to XWALK.md)
+            </metadata>
+        </record>
+        <resumptionToken completeListSize="4000" cursor="0" expirationDate="2018-05-24T14:54:04Z">eyJmIjogIjIwMTYtMDEtMDEiLCAibiI6IDEwMCwgIm0iOiAib2FpX2RjIn0=</resumptionToken>
+    </ListRecords>
+</OAI-PMH>
+```
+
+## Paging Control 
+To use pagination, just make a request using the retrieved resumption token like so. Where verb is either ListRecords or ListIdentifiers. 
```

Comment:
`To use pagination, just make a request using the retrieved resumption token (together with verb ListRecords or ListIdentifiers).`

Have the Example request on a new line.